scripts/install/ontology/eligibility.rdf

```diff
@@ -42,25 +42,16 @@
     <rdfs:comment xml:lang="en-US">The proctor authorization is not required if this property is true</rdfs:comment>
     <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/TAOProctor.rdf#DeliveryEligibility"/>
   </rdf:Property>
-  
+
+  <rdf:Description rdf:about="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#Hidden">
+    <rdf:type rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#WidgetClass"/>
+  </rdf:Description>
+
   <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAODelivery.rdf#ProctorAccessible">
     <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
-    <rdfs:label xml:lang="en-US"><![CDATA[Accessible by Proctor]]></rdfs:label>
-    <rdfs:comment xml:lang="en-US"><![CDATA[Accessible by Proctor]]></rdfs:comment>
-    <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#CheckBox"/>
-    <generis:is_language_dependent rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#False"/>
+    <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#Hidden"/>
     <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/TAODelivery.rdf#Delivery"/>
-    <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/TAODelivery.rdf#CheckMode"/>
     <tao:TAOGUIOrder><![CDATA[90]]></tao:TAOGUIOrder>
   </rdf:Description>
-  <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAODelivery.rdf#CheckMode">
```

Comment:
You can't delete `CheckMode`. Otherwise all the rest checkboxes which use this property will be broken:
![image](https://user-images.githubusercontent.com/11025793/27796915-5cb089ac-6014-11e7-883b-fa0d71720af5.png)